### ISSUE TITLE
Collapsible blocks with "+" are not rendered as an open block

### ISSUE THREAD

**Issue by chme** [squidfunk/mkdocs-material#3313]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I've found a bug and checked that ...

- [X] ... the problem doesn't occur with the `mkdocs` or `readthedocs` themes
- [X] ... the problem persists when all overrides are removed, i.e. `custom_dir`, `extra_javascript` and `extra_css`
- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

Collapsible blocks (admonitions) with a `+` after the `???` token are not rendered as an open block. 

### Expected behaviour

Adding the `+` should render the block as open.

### Actual behaviour

Block is rendered as closed

### Steps to reproduce

You can see this behavior on the official documentation for mkdocs-material: <https://squidfunk.github.io/mkdocs-material/reference/admonitions/#collapsible-blocks>

The following block is not rendered as open:

``` markdown
???+ note

    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla et euismod
    nulla. Curabitur feugiat, tortor non consequat finibus, justo purus auctor
    massa, nec semper lorem quam in massa.
``` 

### Package versions

- Python: `3.9.9`
- MkDocs: `1.2.3`
- Material: `8.0.4`

### Configuration

```yaml
site_name: My Docs
theme:
  name: material

markdown_extensions:
  - admonition
  - pymdownx.details
  - pymdownx.superfences
```

### System information

- Operating system: Linux (Solus OS)
- Browser: Firefox 94

**Comment by squidfunk** [comment]
Thanks or reporting. This is a regression because the `print$` observable will now also emit on first subscription whether the page is in print mode or not. This led to open details elements always being rendered closed. Should be fixed with accfc4ff8.

**Comment by squidfunk** [comment]
Released as part of 8.0.5.
